access/src/main/java/dk/dbc/rawrepo/RawRepoDAOPostgreSQLImpl.java

```diff
@@ -380,7 +380,7 @@ public String getMimeTypeOf(String bibliographicRecordId, int agencyId) throws R
                     return resultSet.getString(1);
                 }
             }
-            throw new RawRepoExceptionRecordNotFound("Trying to find mimetype");
+            throw new RawRepoExceptionRecordNotFound("Trying to find mimetype for recordId " + bibliographicRecordId + ", agencyId " + agencyId);
```

Comment:
Ved godt det er en gammel tekst, men kunne godt tænke mig at "Failed when trying" ville give bedre mening.

Reply:
Hvad med "Failed to find...."